bindings/wasm/crate/src/lib.rs

```diff
@@ -53,8 +53,13 @@ pub fn create_outbound_session(
         .get(receiving_handle)
         .unwrap()
         .set(receiving_instance);
+
     match result {
-        Ok((_, ciphertext_json)) => Ok(ciphertext_json),
+        Ok((session_id, ciphertext_json)) => Ok(vec![
+            JsValue::from_str(&session_id),
+            JsValue::from_str(&ciphertext_json),
+        ]
```

Comment:
Using the Vec here changes the memory structure. Given the size cannot be known at compile time. 

Imagine using a tuple would be easier to use and would avoid the need for boxing of the values.

Reply:
Unfortunately, `wasm-bindgen` doesn't appear to support tuples as return values. https://github.com/rustwasm/wasm-bindgen/issues/122